### ISSUE TITLE
Chisel dev google group

### ISSUE THREAD

**Issue by da-steve101** [ucb-bar/chisel#505]
Just wondering if there can be a chisel-dev google group to discuss what to work on/priorities/bugs
Think its better to keep separate from chisel-users

**Comment by ccelio** [comment]
Good idea, but in my opinion, the chisel-users group is such low utilization, you should feel free to post your discussion there, if it doesn't fit/match into a particular issue or pull request discussion.
